#Custom message for publishing the detected object position
#object_pose.msg
int16 ID
geometry_msgs/Twist pose
